econml/bootstrap.py

```diff
@@ -97,16 +387,40 @@ def call(*args, **kwargs):
 
         def get_mean():
             # for attributes that exist on the wrapped object, just compute the mean of the wrapped calls
-            return proxy(callable(getattr(self._instances[0], name)), name, lambda arr: np.mean(arr, axis=0))
+            return proxy(callable(getattr(self._instances[0], name)), name, lambda arr, _: np.mean(arr, axis=0))
+
+        def get_std():
+            prefix = name[: - len('_std')]
+            return proxy(callable(getattr(self._instances[0], prefix)), prefix,
+                         lambda arr, _: np.std(arr, axis=0))
 
         def get_interval():
             # if the attribute exists on the wrapped object once we remove the suffix,
             # then we should be computing a confidence interval for the wrapped calls
             prefix = name[: - len("_interval")]
 
             def call_with_bounds(can_call, lower, upper):
-                return proxy(can_call, prefix,
-                             lambda arr: (np.percentile(arr, lower, axis=0), np.percentile(arr, upper, axis=0)))
+                def percentile_bootstrap(arr, _):
+                    return np.percentile(arr, lower, axis=0), np.percentile(arr, upper, axis=0)
+
+                def pivot_bootstrap(arr, _):
+                    # TODO: do we want the central estimate to be the average of all bootstrap estimates,
+                    #       or the original estimate over the entire non-bootstrapped population?
+                    est = np.mean(arr, axis=0)
```

Comment:
Again here the point estimate is the original estimate. This should be calling sth not re-doing the point estimate.

Reply:
Fixed.